app/util/fareUtils.js

```diff
@@ -27,6 +27,19 @@ export function mapFares(fares, config) {
   }));
 }
 
+export function fetchFares(itinerary, url) {
+  return fetch(url, {
+    method: 'POST',
+    // eslint-disable-next-line compat/compat
+    headers: new Headers({ 'content-type': 'application/json' }),
+    body: JSON.stringify(itinerary),
+  })
+    .then(response => response.json())
+    .then(data => {
+      return data;
```

Comment:
The `fetch` Promise will also resolve on failed requests, so `data` will just contain a response body with an error message, but not the actual itinerary. It might make sense to catch this case here using `response.ok` and return `null`.

(Don't want to mansplain here, but I'm probably doing just that, sorry.)

Reply:
No worries, I'm always happy for the suggestions! And fixed this :)